app/views/art_items/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if @art_item.blank? %>
+<p>There are not any art item currently in your portfolio.</p>
+<% else %>
+<p>These are the current art item in your portfolio</p>
```

Comment:
Grammar: maybe this is better "These are the art items currently in your portfolio"

Reply:
You're all right, I'll change it to plural. Thanks!